### DISCUSSION TITLE
Speech Recognition

### DISCUSSION THREAD

**Discussion by rjcjunior** [frontendbr/forum#1844] [Perguntas e Respostas]
Olá, pessoal. Alguém conhece alguma lib, api ou algo do tipo para reconhecimento de voz? 
Tenho um input e gostaria a possibilidade das informações virem por voz.

Já ví o web speech api do google mas conhecem algum outro? Ele parece ser compatível somente com o chrome, gostaria de algo para o IE, EDGE, Safari, etc...

**Comment by ojvribeiro** [comment]
Pro IE eu acho bem difícil, mas pros outros você pode tentar o [Annyang](https://github.com/talater/annyang). Ele facilita bastante sua vida ao trabalhar com speech recognition.
